src/com/geekhub/hw2/vehicle/Boat/boatPart/Screw.java

```diff
@@ -0,0 +1,22 @@
+package com.geekhub.hw2.vehicle.Boat.boatPart;
```

Comment:
"boatPart" and "carPart" packages should be moved to parts package as "boat" and "car".
Also package names should always be in lower case.